src/solvers.jl

```diff
@@ -676,12 +732,15 @@ function conicconstraintdata(m::Model)
         append!(J, indices)
         append!(V, tmpelts[indices])
         empty!(tmprow)
+        constr_dual_map[c] = collect(c)
```

Comment:
scalar to array conversion as it is array of arrays, should I do it some other way??